Assets/Scripts/Interaction/Openable.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Interaction.Core;
+using UnityEngine;
+
+namespace Interaction
+{
+    [RequireComponent(typeof(InteractionReceiver))]
+    public class Openable : MonoBehaviour, IInteraction
+    {
+        public bool open;
+        
+        public void Setup(Action<string> listen, Action<string> blocks)
+        {
+            listen("open");
+        }
+
+        public bool Handle(InteractionEvent e)
+        {
+            open = !open;
+            
+            Debug.Log($"{name} is now {(open ? "open" : "closed")}");
```

Comment:
Actual code to open/close in the game world will go here in the future?

Reply:
Yeah, actually working lockers are outside the scope of this